### PR TITLE
Dg4220 patch 1

### PR DESCRIPTION
**Pull request by dg4220**
version bump

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 **Tags:** permissions, section, access, acl, user management, custom roles, content editing, workflow, boston university, bu  
 **Requires at least:** 3.1  
 **Tested up to:** 4.5.3  
-**Stable tag:** 0.9.9  
+**Stable tag:** 0.10.0
```
Comment (DannyCrews) [inline README.md]: Should be 0.1.0

File: bu-section-editing.php [hunk 2]
```diff
@@ -5,7 +5,7 @@
 Author: Boston University (IS&T)
 Author URI: http://sites.bu.edu/web/
 Description: Enhances WordPress content editing workflow by providing section editing groups and permissions
-Version: 0.9.9
+Version: 0.10.0
```
Comment (DannyCrews) [inline bu-section-editing.php]: Should be 0.1.0

File: README.md (hunk 1 above)
Comment (dg4220) [reply README.md]: that doesn't seem right - from 9 to 1 in the same position? It looks like the tags start at 0.3.0 and go up - I almost made it 0.9.10 it such a small update

Comment (dg4220) [reply README.md]: Actually 0.1 is the first https://github.com/bu-ist/bu-section-editing/tree/6d01687032d76d2844367fb48bf8548f4b56db23

Comment (DannyCrews) [reply README.md]: Yeah. It wasn't done right from the start. I misread. Leave as 0.10.0

**Pull request merged**